app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- kindly note that the application should ask for confirmation of email.
so add ` :confirmable` to enable it in the user model (user.rb file). :+1:

Reply:
Alright thank you, I will implement that right away